File: src/main/kotlin/com/skidders/sigma/utils/render/font/common/FontLanguage.java
package com.skidders.sigma.utils.render.font.common;

public enum FontLanguage {
	ENGLISH(new int[] {31, 127, 0, 0}),
	ENGLISH_RUSSIAN(new int[] {31, 127, 1024, 1106});
	
	public final int[] charCodes;
	
	FontLanguage(int[] charCodes) {
		this.charCodes = charCodes;
	}
}

File: src/main/kotlin/com/skidders/sigma/utils/render/font/common/FontLanguage.kt
package com.skidders.sigma.utils.render.font.common

enum class FontLanguage(val charCodes: IntArray) {
    ENGLISH(intArrayOf(31, 127, 0, 0)),
    ENGLISH_RUSSIAN(intArrayOf(31, 127, 1024, 1106))
}